.github/workflows/ci_tests.yaml

```diff
@@ -203,29 +196,39 @@ jobs:
         GATEKEEPER_VERSION=v$(cat GATEKEEPER_VERSION 2>/dev/null || cat VERSION)
         echo "GATEKEEPER_VERSION=${GATEKEEPER_VERSION}" >> ${GITHUB_ENV}
 
+    - name: Deploy operator
+      env:
+        NAMESPACE: gatekeeper-system
+      run: |
+        make download-binaries
```

Comment:
This is already run in the "Download binaries" step. Was it intentional to run it again?

Reply:
Good catch. It was a copy/paste thing from previously--I'll remove it: 
https://github.com/stolostron/gatekeeper-operator/blob/f74ef59a834ff48b22bad6474a5e32315694a028/.github/workflows/ci_tests.yaml#L222